SU2_CFD/src/iteration_structure.cpp

```diff
@@ -1805,9 +1803,7 @@ void CDiscAdjFluidIteration::Preprocess(COutput *output,
             solver[val_iZone][val_iInst][iMesh][HEAT_SOL]->GetNodes()->Set_Solution_time_n();
           }
           if (grid_IsMoving) {
-            for(iPoint=0; iPoint<geometry[val_iZone][val_iInst][iMesh]->GetnPoint();iPoint++) {
-              geometry[val_iZone][val_iInst][iMesh]->node[iPoint]->SetCoord_n();
-            }
+            geometry[val_iZone][val_iInst][iMesh]->nodes->SetCoord_n();
```

Comment:
Usual change 3(? I lost count): As with CVariable, this kind of method now acts on all the points.